homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
Component/BaseEntity use `state_attributes` and platform use `device_state_attributes` and can overwrite stuff from `state_attributes`